osquery/tables/system/ubuntu/deb_packages.cpp

```diff
@@ -179,7 +199,7 @@ QueryData genDebs(QueryContext &context) {
   dpkg_setup(&packages);
   for (int i = 0; i < packages.n_pkgs; i++) {
     struct pkginfo *pkg = packages.pkgs[i];
-    if (pkg->status == pkg->stat_notinstalled) {
+    if (pkg->status == PKG_STAT_NOTINSTALLED) {
```

Comment:
I don't mind `#if define`ing this. Can we also use the `#if LIBDPKG_VERSION < 1017023` logic to set `PKG_STAT_NOTINSTALLED` to a sane value above? Similarly to the conditional logic for setting `FIELD`?

Ideally we should group preprocessor decisions to the "top-of-file"! Mostly so the implementation code stays pretty. :smile_cat: